### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by Rifat8080**
Hello Microverse Linters Pull Request

- Added HTML file with HTML5 semantic tag
- Added CSS file changed color of H1 tag with green
- .gitignore file available to ignore unnecessary files
- HTML and CSS linters standards are available too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,288 @@
...
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (iambenkis) [inline README.md]: - [x] Good job implementing your first project at Microverse, you make a really great work 👏 , however there's  still things you can improve in your README file 👍 
- [x] Kindly make sure broken links or attached to in your file 👍 please fix the logo  link , you can put your own logo, or you can just remove simply it but don't use the `Microverse logo` 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,288 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (iambenkis) [inline README.md]: - [x] Kindly make sure a working link is attached to this `MIT` link 😄 , 
- [x] Also kindly remove this NOTE it should not be a part of the README file 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,288 @@
...
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```
Comment (iambenkis) [inline README.md]: - [x] By using the README provided by Microverse that have everything to showcase your abilities, skills etc, in terms of technical knowledge. You just need to customize that template according to your project requirement. Take a look at the below screenshot for more detail.

I noticed that you did not complete sections ( project_name,  description, built with , key features, etc ) of the README file , Kindly complete your documentation 👍 


Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Rifat8080 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project, but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Rifat8080, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have changes as requested by the previous reviewer, kindly make them then submit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Rifat8080 — COMMENTED** [review]
Change of contents!

- Added built in languages
- Removed Microverse logo
- Given Author LinkedIn, Tweeter, Github
- MIT licensed added

File: README.md [hunk 4]
```diff
@@ -1 +1,232 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> It is a project build with HTML and CSS according to linters standards.
```
Comment (V-Blaze) [inline README.md:56]: - [OPTIOANL] You have done a great customizing each section of your readme file :+1: :100: , To make your readme file look more professional, I'll suggest you go through the readme once more and remove all the placeholder comments. This  `It is a project build with HTML and CSS according to linters standards.` is an example, I'll also point out the other ones below. :+1:

File: README.md [hunk 5]
```diff
@@ -1 +1,232 @@
...
+
+**[Hello Microverse Project]** is a trial educational project coursework of Microverse.
+
+## 🛠 Built With HTML And CSS <a name="built-with"></a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (V-Blaze) [inline README.md:66]: - [OPTIONAL] This is a placeholder comment, please remove this. :smile: :+1:

File: README.md [hunk 6]
```diff
@@ -1 +1,232 @@
...
+
+- [Live Demo Link](https://github.com/Rifat8080/Hello-Microverse-Project)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (V-Blaze) [inline README.md:88]: - [OPTIONAL] This is also a placeholder comment, please remove this. :smile: :+1:

File: README.md [hunk 7]
```diff
@@ -1 +1,232 @@
...
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (V-Blaze) [inline README.md:172]: - [OPTIONAL] This is also a placeholder comment, please remove this. :smile: :+1:

File: README.md [hunk 8]
```diff
@@ -1 +1,232 @@
...
+- Twitter: [@mdmahadirifat98](https://twitter.com/mdmahadirifat98)
+- LinkedIn: [md-mahadi-hasan-rifat-8b7851265](https://www.linkedin.com/in/md-mahadi-hasan-rifat-8b7851265/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (V-Blaze) [inline README.md:186]: - [OPTIONAL] This is also a placeholder comment, please remove this. :smile: :+1:

File: README.md [hunk 9]
```diff
@@ -1 +1,232 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (V-Blaze) [inline README.md:208]: - [OPTIONAL] This is also a placeholder comment, please remove this. :smile: :+1:

File: README.md [hunk 10]
```diff
@@ -1 +1,232 @@
...
+
+If you like this project let us know with a DM!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (V-Blaze) [inline README.md:218]: - [OPTIONAL] This is also a placeholder comment, please remove this. :smile: :+1:

**Review by V-Blaze — COMMENTED** [review]
Hi @Rifat8080  , 

# Approved 🟢 🟢 

## NB: According to Microverse requirement, you are not supposed to merge a pull request until it has been reviewed and approved by a code reviewer, Please in your subsequent projects, do not merge an unapproved pull request. Happy Coding!  :+1: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Implemented the changes requested by the previous review💯
-  Readme looks professional 👍
-  Pull request is descriptive🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**